policyengine_us/variables/gov/states/ri/tax/income/credits/property_tax/ri_property_tax_credit.py

```diff
@@ -0,0 +1,29 @@
+from policyengine_us.model_api import *
+
+
+class ri_property_tax_credit(Variable):
```

Comment:
make a ri_property_tax_credit_eligible.py 
this determines if the head or spouse are over 65 or are disabled

Reply:
you can use 
`age_head`, `age_spouse`, `head_is_disabled`, `spouse_is_disabled` for the eligibility